### PR TITLE
DDF-2296: Fixed local resource query to include anytag search

### PR DESCRIPTION
**Pull request by rzwiefel**
*This is a re-pullrequest since catalogframework rework*

#### What does this PR do?
Adds back the anytag search into the catalog framework queries while getting resources

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@djblue 
@ryeats 
@stustison 
@kcwire 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@pklinef

### CODE REVIEW

**Comment by rzwiefel** [comment]
Bamboo is a dirty rotten liar (probably). Just did a full build locally and it was fine.

File: catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java [hunk 1]
```diff
@@ -460,10 +461,16 @@ public Response getDocument(@Encoded @PathParam("sourceid") String encodedSource
                 if (transformerParam != null) {
                     transformer = transformerParam;
                 }
-                Filter filter = getFilterBuilder().attribute(Metacard.ID)
+                Filter idFilter = getFilterBuilder().attribute(Metacard.ID)
                         .is()
                         .equalTo()
                         .text(id);
+                Filter tagsFilter = getFilterBuilder().attribute(Metacard.TAGS)
```
Comment (pklinef) [inline catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java]: Why did you need to change the endpoint code?  Shouldn't the catalog framework be taking care of this?

Comment (coyotesqrl) [inline catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java]: Definitely seems like it's happening twice now.

**Comment by rzwiefel** [comment]
misinformed, no need for this PR because of @coyotesqrl  change

File: catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java (hunk 1 above)
Comment (rzwiefel) [inline catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java]: Took this out.

**Comment by pklinef** [comment]
👍

**Comment by coyotesqrl** [comment]
👍

**Comment by kcwire** [comment]
👍

**Comment by jlcsmith** [comment]
Need to hold off merging this until the Federation itest failures are addressed
